site/content/contribute/server/feature-flags.md

```diff
@@ -72,3 +72,17 @@ There are no hard rules on when a feature flag should be used. It is left up to
 ## Examples of feature flags
 
 < add some examples when we create them >
+
+## FAQ
```

Comment:
Let me know if any other common Q&A should be added here.

Reply:
What's the environment variable to set a feature flag?
It's `MM_FEATUREFLAGS_<myflag>`

Reply:
I added a comment [here](https://community.mattermost.com/core/pl/6tk95nd4otbqx8xcb586webt6c) that depending on the answers on the channel may be a nice question to add here.